tools/travis/production-helper

```diff
@@ -87,7 +87,7 @@ fi
 echo; echo "Now confirming all the RabbitMQ queue processors are correctly registered!"; echo
 # These hacky shell scripts just extract the sorted list of queue processors, running and expected
 supervisorctl status | cut -f1 -dR | cut -f2- -d: | grep events | cut -f1 -d" " | cut -f3- -d_ | cut -f1 -d- | sort -u > /tmp/running_queue_processors.txt
-su zulip -c /home/zulip/deployments/current/scripts/lib/queue_workers.py | grep -v ^test$ > /tmp/expected_queue_processors.txt
+su zulip -c /home/zulip/deployments/current/scripts/lib/queue_workers.py | grep -v ^test$ | sort -u > /tmp/expected_queue_processors.txt
```

Comment:
OK; next time put that in it's own commit with a commit message explaining why.  As a project, we really value each change being reviewed for correctness.